modules/openapi-generator/src/main/resources/cpp-qt-client/api-body.mustache

```diff
@@ -397,7 +397,7 @@ void {{classname}}::{{nickname}}({{#allParams}}{{#required}}const {{{dataType}}}
         }
         fullPath.append(paramString);
         {{/isPrimitiveType}}{{#isPrimitiveType}}
-        fullPath.append(QUrl::toPercentEncoding("{{baseName}}")).append(querySuffix).append(QUrl::toPercentEncoding(::{{cppNamespace}}::toStringValue({{paramName}}{{^required}}.value(){{/required}})));
+        fullPath.append(QUrl::toPercentEncoding("{{baseName}}")).append(querySuffix).append(QUrl::toPercentEncoding({{paramName}}{{^required}}.stringValue(){{/required}}));
```

Comment:
why did you removed the cppNamespace ?

Reply:
The namespace was needed  for the toStringValue method. Now I call the stringValue member function of the parameter itself. For that call I did not need to specify the namespace (if I am not mistaken).

Reply:
Ok that makes sense!